### PR TITLE
Refactor to send directly to secure message api

### PR DESCRIPTION
**Pull request by dcdarrell9**
Removed usage of frontstage-api calls to secure-message and call directly

### CODE REVIEW

**Comment by dcdarrell9** [comment]
[Refactor frontstage to send messages using the secure-message API directly (5)](https://trello.com/c/DGyASdWF/177-refactor-frontstage-to-send-messages-using-the-secure-message-api-directly-5)

File: frontstage/views/secure_messaging/create_message.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 from frontstage.common.api_call import api_call
 from frontstage.common.session import SessionHandler
```
Comment (MIslam92) [inline frontstage/views/secure_messaging/create_message.py]: Forgot to remove these unused imports?

File: frontstage/controllers/conversation_controller.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import json
```
Comment (MIslam92) [inline frontstage/controllers/conversation_controller.py]: This import is not used either?

File: frontstage/views/secure_messaging/create_message.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 from frontstage.common.api_call import api_call
```
Comment (MIslam92) [inline frontstage/views/secure_messaging/create_message.py]: Unused import

File: frontstage/views/secure_messaging/create_message.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
...
 from frontstage.common.session import SessionHandler
+from frontstage.controllers import conversation_controller
 from frontstage.exceptions.exceptions import ApiError
```
Comment (MIslam92) [inline frontstage/views/secure_messaging/create_message.py]: Unused import again

**Comment by dcdarrell9** [comment]
The `make test` must not be running the lint, my bad, I'll see if I can get it running with that

File: frontstage/controllers/conversation_controller.py [hunk 5]
```diff
@@ -72,10 +72,11 @@ def send_message(message_json):
...
         response = session.post(url, headers=headers, data=message_json)
         response.raise_for_status()
-    except HTTPError as ex:
+        logger.info("Message sent successfully")
+        return response.json()
+    except HTTPError:
         logger.exception("Message sending failed due to API Error")
-        raise ApiError(ex.response)
-    logger.info("Message sent successfully")
+        raise ApiError(response)
```
Comment (MIslam92) [inline frontstage/controllers/conversation_controller.py]: `raise ApiError(response)` throws the classic `Local variable might be referenced before assignment` warning so might be worth setting response to none prior to the try/except?

Comment (insacuri) [reply frontstage/controllers/conversation_controller.py]: I think you can put the post outside the try/except to fix the issue.  I think that's what we do in response-operations-ui but you'll have to double check

File: frontstage/controllers/conversation_controller.py [hunk 6]
```diff
@@ -72,10 +72,11 @@ def send_message(message_json):
     try:
         response = session.post(url, headers=headers, data=message_json)
         response.raise_for_status()
-    except HTTPError as ex:
+        logger.info("Message sent successfully")
```
Comment (insacuri) [inline frontstage/controllers/conversation_controller.py]: Could you put a ```python

Comment (dcdarrell9) [reply frontstage/controllers/conversation_controller.py]: Line 64 in create_message.py gives us the message id and party id, so I think that should be fine 👍

Comment (insacuri) [reply frontstage/controllers/conversation_controller.py]: It's not good enough, lots of the time when you're debugging an issue and looking through logs, you filter on an ID to begin with, see what messages you get then go from there.  And we want to make that part as easy as possible whereever we can

Think about in SDX, pretty much all of the lines have a tx_id on them and when you search in splunk, it's really easy to see the whole flow.  If you want anything that hasn't got that id then it's more of a pain to find generally

**Comment by MIslam92** [comment]
@dcdarrell9 Squash and merge?

**Pull request merged**